Sources/MapboxMaps/Foundation/MapView+Lifecycle.swift

```diff
@@ -0,0 +1,89 @@
+import Foundation
+import UIKit
+
+@available(iOSApplicationExtension, unavailable)
+extension MapView {
```

Comment:
Let's put this in the main MapView implementation so that we can keep more of its interface private.

Reply:
Moved it back